projects/auth0-angular/src/lib/auth.client.ts

```diff
@@ -23,7 +23,10 @@ export class Auth0ClientFactory {
       auth0Client: {
         name: useragent.name,
         version: useragent.version,
-      },
+        env: {
+          'angular/core': VERSION.full
+        }
+      } as any,
```

Comment:
Will need to drop this once https://github.com/auth0/auth0-spa-js/pull/913 is merged

Reply:
It has been merged now.

Reply:
Thanks. I forgot the fact that it will also need a release. So will hold of merging here.

Reply:
Or maybe we should merge as is for now. As it should work, and might be a bit weird to cut a release in SPA-JS only for this.